cosmo_tester/test_suites/test_blueprints/test_existing_vm.py

```diff
@@ -99,19 +104,28 @@ def create_keypair_and_copy_to_manager(self,
         fabric.api.put(local_path=key_file,
                        remote_path=remote_key_path)
 
+    def get_nics_by_network_name(self, network_name, neutron_client):
+        nets = neutron_client.list_networks()
+
+        for net in nets['networks']:
+            if net['name'] == network_name:
+                return [{'net-id': net['id']}]
+
```

Comment:
I'd probably have the "raise" line here, rather than outside of the function, I just find it clearer that way, but this obviously doesn't matter much in this case.